index.html

```diff
@@ -18,7 +18,7 @@
     <section class="colors">
```

Comment:
We noticed the text associated with each box wasn't aligned properly. We discussed as a team and plan to refactor our HTML to create containers for each box/text combo.